lib/policies/terminate/terminate.js

```diff
@@ -1,3 +1,10 @@
 module.exports = (params) =>
-  (req, res, next) =>
-    res.status(params.statusCode).send(params.message);
+  (req, res, next) => {
+    let payload;
```

Comment:
I wouldn't bother creating a `let` variable on the top — I'd construct the payload on the fly.

I always sweat when I see a `let` variable :)

Reply:
LOL, that's fine. I was following the lead of similar situations I saw throughout the code (for example https://github.com/ExpressGateway/express-gateway/blob/master/lib/policies/basic-auth/basic-auth.js#L11) and avoid repeating the `res.status().send()` line, but I'm happy to use whatever style you prefer.

Reply:
I understand you're confusion. This code base has been written by multiple persons with the passing of the time before it ended up to me — and I have a completely different mindset compared to the guys that were working on this before.

Reply:
Fixed the style in ac8f74d